### ISSUE TITLE
Objects can have related files

### ISSUE THREAD

**Issue by jcoyne** [projecthydra-labs/hydra-pcdm#62]

**Comment by jcoyne** [comment]
Blocked by projecthydra/active_fedora#788

**Comment by terrellt** [comment]
Unblocked.

**Comment by elrayle** [comment]
This behavior is inherited from Hydra::PCDM.  Tests were added with PR https://github.com/projecthydra-labs/hydra-works/pull/96/files.  The ability to add related objects is now provided by services.  See PR https://github.com/projecthydra-labs/hydra-works/pull/106.  Tests for related objects moved from model tests to service tests with this PR.

**Comment by jcoyne** [comment]
Won't fix. We should use ina:describes for the fits.xml and a file that its describes.
